src/Item.hpp

```diff
@@ -57,7 +57,7 @@ class Item
 	, _path(folder)
 	, _sequence(sequence)
 	{
-		_path /= sequence.getStandardPattern();
+		_path /= sequence.getPrintfPattern();
```

Comment:
In that case, they should ask for the printf style value explicitly instead of changing the default value in the engine.